### ISSUE TITLE
Bug with searching rooms to add to a space

### ISSUE THREAD

**Issue by chagai95** [vector-im/element-android#5954]
### Steps to reproduce

The conversations section is not updated only the rooms.
https://aarenet-my.sharepoint.com/:v:/p/chagai_friedlander/EVlDtMIWTJJJhmy_mNrtSzEBW2JJ6I4W69KYH3Ggo38O3g

### Your phone model

Xiaomi 11

### Operating system version

Android 11 RKQ1.200826.002, MIUI 12.5.6.0 (RKQEUXM)

### Application version and app store

private fork

### Homeserver

chagai.website

### Will you send logs?

No

**Comment by MadLittleMods** [comment]
@chagai95 Please update the issue description with what's missing and what you expect. It's hard to follow what's wrong in the video.

**Comment by chagai95** [comment]
I find it really unintuitive to answer these questions... There's nothing missing - it's just something that's not working the way it should:
When you try searching it does not search (update the results) 

and what "I expect" is for it "to work" so it's really weird to just answer the questions... Or are those the answers you are looking for?

I'm missing:
A working feature.

I expect:
The feature to work

The feature being search, and more specifically the section ""conversations" when you search for rooms to add to a space...

@kittykat maybe you can help? 🙈
